### ISSUE TITLE
Italian translation of admin interface

### ISSUE THREAD

**Issue by DavideTriso** [sulu/sulu#4902]
Hello,
do you already provide Italian translations for the admin interface?
If not I would like to contribute to sulu project and provide them.

Can you please give me some advice on how to contribute?

**Comment by alexander-schranz** [comment]
@DavideTriso We currently only provide DE and EN as admin interface language.
We are still searching for a online tool to setup for translations to make it easier for the community to contribute translations. And hope that we also can then provide FR and NL again. We would be happy if somebody would work to provide IT.

Currently the necessary files for translations are the `admin.en.json` or if you prefer german the `admin.de.json` in every bundles Resources/translations folder. If you want you can begin translating this files and when we setup the tool we could import them into it. Until you can use them by copy it in your projects translations folder.

That you can select IT as system language you need to add it also to the configuration of the sulu_core bundle e.g.:

```yaml
sulu_core:
    locales:
        de: Deutsch
        en: English
        it: Italiano
    translations:
        - de
        - en
        - it
    fallback_locale: en
```

**Comment by DavideTriso** [comment]
Ok, I will start working on that as soon as possible.

**Comment by cirykpopeye** [comment]
@alexander-schranz I saw Sylius uses Crowdin for its translations. https://crowdin.com/project/sylius Works really well imo, maybe something for Sulu?

**Comment by alexander-schranz** [comment]
@cirykpopeye yes we had also a look on some providers. I'm personally not happy with crowdin, last time I tried to contribute to sylius it was just a mess with the translations and versions. Know that others tried lingohub not sure if that is maybe better, but think we will try some providers in the next time to find which fits best for sulu.

**Comment by danrot** [comment]
The problem with lingohub is that they do not support any crowd translation, so I think we'll have to go with crowdin.

**Comment by danrot** [comment]
You should be able to translate the `sulu/sulu` project on https://sulu.crowdin.com/, already created the italian language there. #4952 will be merged soon, as soon as this has happened, you can download the italian version using `bin/console sulu:admin:download-language it`.

**Comment by danrot** [comment]
Fixed in #4952 :tada:
